spec/request/apps_spec.rb

```diff
@@ -2104,101 +2104,139 @@
-                                                            'start' => { 'href' => "#{link_prefix}/v3/apps/#{app_model.guid}/actions/start", 'method' => 'POST' },
-                                                            'stop' => { 'href' => "#{link_prefix}/v3/apps/#{app_model.guid}/actions/stop", 'method' => 'POST' },
-                                                            'revisions' => { 'href' => "#{link_prefix}/v3/apps/#{app_model.guid}/revisions" },
-                                                            'deployed_revisions' => { 'href' => "#{link_prefix}/v3/apps/#{app_model.guid}/revisions/deployed" },
-                                                            'features' => { 'href' => "#{link_prefix}/v3/apps/#{app_model.guid}/features" },
-                                                        }
-                                                    })
-
-      event = VCAP::CloudController::Event.last
-      expect(event.values).to include({
-                                          type: 'audit.app.start',
-                                          actee: app_model.guid,
-                                          actee_type: 'app',
-                                          actee_name: 'app-name',
-                                          actor: user.guid,
-                                          actor_type: 'user',
-                                          actor_name: user_email,
-                                          actor_username: user_name,
-                                          space_guid: space.guid,
-                                          organization_guid: space.organization.guid,
-                                      })
-    end
+    context 'app lifecycle is buildpack' do
+      let!(:droplet) do
+        VCAP::CloudController::DropletModel.make(
+          :buildpack,
+          app: app_model,
+          state: VCAP::CloudController::DropletModel::STAGED_STATE
+        )
+      end
 
-    context 'telemetry' do
-      it 'should log the required fields when the app starts' do
+      before do
         app_model.lifecycle_data.buildpacks = ['http://example.com/git']
         app_model.lifecycle_data.stack = stack.name
         app_model.lifecycle_data.save
-
-        droplet = VCAP::CloudController::DropletModel.make(:buildpack, app: app_model, state: VCAP::CloudController::DropletModel::STAGED_STATE)
         app_model.droplet = droplet
         app_model.save
+      end
 
-        Timecop.freeze do
-          expected_json = {
-            'telemetry-source' => 'cloud_controller_ng',
-            'telemetry-time' => Time.now.to_datetime.rfc3339,
-            'start-app' => {
-              'api-version' => 'v3',
-              'app-id' => Digest::SHA256.hexdigest(app_model.guid),
-              'user-id' => Digest::SHA256.hexdigest(user.guid),
+      context 'it starts the app' do
```

Comment:
Super minor/nitpicky, but it feels a little odd to have contexts starting with the phrase "it", given that there are `it` blocks. I Would it feel reasonable to change these to `starting an app`, `stopping an app`, and `restarting an app`?

Reply:
Perfect, thanks for being so responsive. Happy to help unblock you all, thanks for the PR!